apps/src/templates/SafeMarkdown.jsx

```diff
@@ -17,6 +17,7 @@ import rehypeReact from 'rehype-react';
 import defaultSanitizationSchema from 'hast-util-sanitize/lib/github.json';
 
 import externalLinks from './plugins/externalLinks';
+import '../../style/curriculum/documentation_tables.scss';
```

Comment:
Is this still needed?

Reply:
After a lot of trial/error and experimentation, I was copying from [this file](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/sectionProgress/progressTables/ProgressTableContentView.jsx#L16) and when I delete the import statement from SafeMarkdown, the styling goes away.  SO, in conclusion, it feels like I need the statement, I am not 100% sure why though... I am going to add this to our 1-1 conversation.

Reply:
Bethany wondering: could we move this to a different file?